chrony/map.jinja

```diff
@@ -1,22 +1,24 @@
-{## start with defaults from defaults.yaml ##}
-{% import_yaml slspath+"/defaults.yaml" as defaults %}
-{% import_yaml slspath+"/osmap.yaml" as osmap %}
+# -*- coding: utf-8 -*-
+# vim: ft=jinja
 
-{% set chrony = salt['grains.filter_by'](
-    defaults,
-    merge=salt['grains.filter_by'](
-        osmap,
-        grain='os_family',
-        merge=salt['pillar.get']('chrony', {}),
-        ),
-    base='chrony'
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{#- Start imports as #}
+{%- import_yaml tplroot ~ "/defaults.yaml" as default_settings %}
+{%- import_yaml tplroot ~ "/osfamilymap.yaml" as osfamilymap %}
+{%- import_yaml tplroot ~ "/osmap.yaml" as osmap %}
+{%- import_yaml tplroot ~ "/osfingermap.yaml" as osfingermap %}
+
+{%- set defaults = salt['grains.filter_by'](default_settings,
+    default='chrony',
+    merge=salt['grains.filter_by'](osfamilymap, grain='os_family',
+      merge=salt['grains.filter_by'](osmap, grain='os',
+        merge=salt['grains.filter_by'](osfingermap, grain='osfinger',
+          merge=salt['pillar.get']('chrony:lookup', default={})
+        )
+      )
+    )
 ) %}
 
-{# Debian distros check /etc/default/rcS to determine UTC setting #}
-{% if grains['os_family'] == "Debian" %}
-  {% if salt['cmd.run']('grep UTC=no /etc/default/rcS', output_loglevel='quiet') %}
-     {%- if chrony['otherparams'] and 'rtconutc' in chrony['otherparams'] %}
-         {% do chrony['otherparams'].remove('rtconutc') %}
-     {%- endif %}
-  {% endif %}
-{% endif %}
+{#- Merge the chrony pillar #}
+{%- set chrony = salt['pillar.get']('chrony', default=defaults, merge=True) %}
```

Comment:
[Likewise](https://github.com/saltstack-formulas/chrony-formula/pull/16/files#r279147483), these two lines have been updated.

Reply:
This has been updated to `config.get`